client/jetpack-cloud/sections/partner-portal/issue-multiple-licenses-form/index.tsx

```diff
@@ -22,49 +29,54 @@ export default function IssueMultipleLicensesForm( {
 }: AssignLicenceProps ) {
 	const translate = useTranslate();
 	const dispatch = useDispatch();
-	const { data: allProducts, isLoading: isLoadingProducts } = useProductsQuery( {
-		select: selectAlphaticallySortedProductOptions,
-	} );
+
+	const { data: allProducts, isLoading: isLoadingProducts } = useProductsQuery();
 
 	const bundles =
-		allProducts?.filter( ( { family_slug } ) => family_slug === 'jetpack-packs' ) || [];
+		allProducts?.filter(
+			( { family_slug }: { family_slug: string } ) => family_slug === 'jetpack-packs'
+		) || [];
 	const products =
-		allProducts?.filter( ( { family_slug } ) => family_slug !== 'jetpack-packs' ) || [];
+		allProducts?.filter(
+			( { family_slug }: { family_slug: string } ) => family_slug !== 'jetpack-packs'
+		) || [];
 
 	// If the user comes from the flow for adding a new payment method during an attempt to issue a license
 	// after the payment method is added, we will make an attempt to issue the chosen license automatically.
 	const defaultProductSlugs = getQueryArg( window.location.href, 'products' )
 		?.toString()
 		.split( ',' );
 
-	const [ selectedProductSlugs, setSelectedProductSlugs ] = useState( defaultProductSlugs ?? [] );
```

Comment:
I moved the `selectedProductSlugs` into redux so that they could be accessed in the `TotalCost` component.